src/components/Header/index.js

```diff
@@ -10,18 +11,17 @@ function Header() {
       pad={{ vertical: 'xsmall', horizontal: 'medium' }}
       justify="between"
     >
-      <Link to="/" style={{ textDecoration: 'none' }}>
-        <Text weight="bold" color="dark-1">
-          HPE Developer
-        </Text>
-      </Link>
+      <ButtonLink label="HPE Developer" to="/" />
       <Box direction="row" gap="medium" justify="center">
```

Comment:
These don't resize well when the window is small. You should probably at least add `wrap` here.

Reply:
Yes, I agree with you. I added wrap and asked Bhim for a mobile design for the header.